src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/GaugeWave.cpp

```diff
@@ -41,13 +40,19 @@ DataType gauge_wave_deriv_h(const tnsr::I<DataType, Dim>& x, const double t,
 namespace gr {
 namespace Solutions {
 template <size_t Dim>
-GaugeWave<Dim>::GaugeWave(const double amplitude,
-                          const double wavelength) noexcept
+GaugeWave<Dim>::GaugeWave(const double amplitude, const double wavelength,
+                          const OptionContext& context)
     : amplitude_(amplitude), wavelength_(wavelength) {
-  ASSERT(amplitude > 0.0,
-         "Amplitude must be non-negative. Given amplitude: " << amplitude_);
-  ASSERT(wavelength > 0.0,
-         "Wavelength must be non-negative. Given wavelength: " << wavelength_);
+  if (abs(amplitude) >= 1.0) {
+    PARSE_ERROR(context,
+                "Amplitude must be less than one. Given amplitude: "
```

Comment:
Why does the amplitude have to be less than 1? I thought you could make the coordinates move by any amount you want

Reply:
Looking at the documentation, det(g) = -H^2, so it looks like we should avoid H=0.  I have not done any numerical testing of the A >= 1 regime, so it is possible there is some subtlety here that I am missing.

Reply:
@geoffrey4444 did you see this comment?

Reply:
@geoffrey4444 please look at the comment

Reply:
I'm OK with this as-is, just wanted @wthrowe to think about it...Please accept my apologies for taking so long to reply! I missed the notifications that I needed to respond